src/test/java/com/conveyal/taui/TestUtils.java

```diff
@@ -0,0 +1,175 @@
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class TestUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(TestUtils.class);
+
+    /**
+     * Parse a json string into an unmapped JsonNode object
+     */
+    public static JsonNode parseJson(String jsonString) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        return mapper.readTree(jsonString);
+    }
+
+    /**
+     * Helper to return the relative path to a test resource file
+     *
+     * @param fileName
+     * @return
+     */
+    public static String getResourceFileName(String fileName) {
+        return String.format("./src/test/resources/%s", fileName);
+    }
+
+    /**
+     * Recursively removes all specified keys.  This is used to make reliable snapshots.
+     */
+    public static void removeKeysAndValues (JsonNode json, String[] keysToRemove) {
+        // remove key/value pairs that are likely to have different values each test run
+        for (String key : keysToRemove) {
+            if (json.has(key)) {
+                ((ObjectNode) json).remove(key);
+            }
+        }
+
+        // iterate through either the keys or array elements of this JsonObject/JsonArray
+        if (json.isArray() || json.isObject()) {
+            for (JsonNode nextNode : json) {
+                removeKeysAndValues(nextNode, keysToRemove);
+            }
+        }
+    }
+
+    /**
+     * Recursively removes all dynamically generated keys in order to perform reliable snapshots
+     */
+    public static void removeDynamicValues(JsonNode json) {
+        removeKeysAndValues(json, new String[]{"_id", "createdAt", "nonce", "updatedAt"});
+    }
+
+    /**
+     * An assertion to make sure that a given objectId is present in at least one of the objects in a response that is
+     * supposed to contain a list of objects.
+     *
+     * This function should be passed a rest assured object built up until the point of calling .then()
+     */
+    public static void assertObjectIdInResponse(Response response, String objectId) {
+        assertObjectIdInResponsePresence(response, objectId, true);
+    }
+
+    /**
+     * An assertion to make sure that a given objectId is not present in any of the objects in a response that is
+     * supposed to contain a list of objects
+     *
+     * This function should be passed a rest assured object built up until the point of calling .then()
+     */
+    public static void assertObjectIdNotInResponse(Response response, String objectId) {
+        assertObjectIdInResponsePresence(response, objectId, false);
+    }
+
+    /**
+     * Helper method to do the parsing and checking of whether an object with a givenn ObjectId is present or not in a
+     * response that contains a list of objects
+     */
+    public static void assertObjectIdInResponsePresence(Response response, String objectId, boolean shouldBePresent) {
```

Comment:
I would turn this into a function that returns a boolean and then run assertions on that boolean.